src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        String color = new ColorSupplier().getRandomColor().toString();
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/197787305-c8ffeaf2-fecd-41ab-96ad-0c4d4238d4ea.png)

if the mentor finds an error from the [checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery) they can reject it immediately and don't check until you fix all the problems from the checklist
**You have to read this checklist and fix common mistakes before you submit your Pull Request**

Reply:
I have fixed it.